webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -44,7 +45,13 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
           <div className={styles.rowPath}>
             <button className={styles.rowToggler} onClick={toggleIsShownState}>
               <Icon icon={isShown ? ChevronDown : ChevronRight} />
-              <span className={styles.pathText}>{path}</span>
+              <Tooltip
+                content={path}
+                placement="bottom-start"
+                delay={[1000, 0]}
```

Comment:
Do we want this delay in some kind of variable?

Reply:
You're right. I lifted it from the experiments table headers and thought the same but then forgot about it. I'll do a follow up for this.